rero_mef/agents/viaf/api.py

```
@@ -382,6 +388,59 @@ def update_online(self, dbcommit=False, reindex=False):
+        # Create/update the target VIAF record
+        target_data.pop("redirect_from", None)
+        target_data = add_md5(target_data)
+
+        try:
+            new_record, action = self.create_or_update(
+                data=target_data,
+                dbcommit=dbcommit,
+                reindex=reindex,
+                test_md5=True,
+            )
+        except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)